docs/cmd/tkn_taskrun_delete.md

```diff
@@ -0,0 +1,46 @@
+## tkn taskrun delete
+
+Delete a taskrun in a namespace
+
+***Aliases**: rm*
+
+### Usage
+
+```
+tkn taskrun delete
+```
+
+### Synopsis
+
+Delete a taskrun in a namespace
+
+### Examples
+
+
+# Delete a TaskRun of name 'foo' in namespace 'bar'
```

Comment:
I think there is no need to have a H1 for `Delete a TaskRun of name 'foo' in namespace 'bar'`.

Reply:
Thank you for your review!!
another docs also have a H1 these. https://github.com/tektoncd/cli/blob/master/docs/cmd/tkn_resource_describe.md
I think we need make a rule about document format.
(after that, we need correct these!)

Reply:
Agree.